### PR TITLE
IE8 doesn't support console.log.apply

### PR DESCRIPTION
**Pull request by tracer99**

### CODE REVIEW

File: ext/dev.js [hunk 1]
```diff
@@ -117,10 +117,14 @@
 			}
 		},
 		_logger:function(type, arr){
-			if(console.log.apply){
-				console[type].apply(console, arr);
-			} else {
+			if (typeof console.log == "object") {
```
Comment (matthewp) [inline ext/dev.js]: What is this case for?

Comment (tracer99) [inline ext/dev.js]: This is feature detection, specifically for IE8. console.log is an object in IE8 but a function in all other modern browsers. console.log.apply does not exist in IE8

Comment (matthewp) [inline ext/dev.js]: If it's an object then how do you use it? Won't `console[type](arr);` throw?

Comment (tracer99) [inline ext/dev.js]: Nope. Just tried it in IE8.

Comment (matthewp) [inline ext/dev.js]: Can you explain though?  If `type` is "log" and you are saying that console.log is an object, you can't call an object, it should throw. I don't doubt that your code is correct, I just want to understand it.

Comment (matthewp) [inline ext/dev.js]: Ok, I researched this a bit myself and it seems that console.log is an object but you can call it like a function. Really bizarre.  However I also found out that `console` doesn't exist until you open the dev tools. So we should really be checking typeof console as well.

Comment (tracer99) [inline ext/dev.js]: Trying to explain how IE8 works half of the time is an exercise in frustration.

Comment (tracer99) [inline ext/dev.js]: Sure. I'll amend the code

**Comment by matthewp** [comment]
Thank you!

**Pull request merged**